ashlar/models.py

```diff
@@ -58,6 +58,7 @@ class Record(AshlarModel):
     occurred_to = models.DateTimeField()
 
     geom = models.PointField(srid=settings.ASHLAR['SRID'])
+    nominatim = models.CharField(max_length=200, null=True, blank=True)
```

Comment:
I think it would be good to make the name of this field more general, since it's not guaranteed that it will always be populated from Nominatim; that just happens to be how we've chosen to do it in DRIVER. Perhaps `location_text`?